lib/libzfs/libzfs_import.c

```diff
@@ -88,6 +88,7 @@ typedef struct name_entry {
 	char			*ne_name;
 	uint64_t		ne_guid;
 	uint64_t		ne_order;
+	uint64_t		ne_labels;
```

Comment:
I had the same thought and went back and forth between `ne_labels` and `ne_nr_labels`.  `ne_nr_labels` is a little clearer but I didn't like the look of the extra `_`.  We could just add a comment to the struct.